core/src/main/java/me/mykindos/betterpvp/core/utilities/events/GetPlayerPropertyEvent.java

```diff
@@ -0,0 +1,24 @@
+package me.mykindos.betterpvp.core.utilities.events;
+
+import lombok.Getter;
+import lombok.Setter;
+import me.mykindos.betterpvp.core.framework.events.CustomEvent;
+import org.bukkit.entity.Player;
+
+@Getter
+public class GetPlayerPropertyEvent extends CustomEvent {
```

Comment:
Not sure I agree with the choice of name here, you aren't really getting a players 'property',

Perhaps something like GetPlayerRelationship that just takes Player and Target as arguments, but stores a single EntityProperty inside the event

i.e. 

GetPlayerRelationshipEvent event = UtilServer.callEvent(new GetPlayerRelationshipEvent(player, target));
if(event.getRelationship() == EntityProperty.FRIENDLY) {
   // work
}

Reply:
Implemented. Sometimes naming something is hard.